interfaces/wizard/static/style.night.css

```diff
@@ -0,0 +1,120 @@
+body {
```

Comment:
I see quite some overlap, can we not combine the login/wizard/config dark skins?

Reply:
We could - which directory should the combined skin go in?

Reply:
The `staticcfg`.

Reply:
Also might want to use how glitter does loading the night theme based on config setting (dark or auto based on os) rather than assume by os only.